### PR TITLE
build proxy plugins on el9

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

**Comment by evgeni** [comment]
(draft, I am still testing whether things actually build)

**Comment by evgeni** [comment]
- smart_proxy_container_gateway needs https://github.com/Katello/smart_proxy_container_gateway/pull/30

File: package_manifest.yaml [hunk 1]
```diff
@@ -192,6 +194,8 @@ plugin_packages:
     repoclosure_target_repos:
       el8:
         - "el8-foreman-plugins-{{ foreman_version }}-staging"
+      el9:
+        - "el9-foreman-plugins-{{ foreman_version }}-staging"
     repoclosure_lookaside_repos:
```
Comment (evgeni) [inline package_manifest.yaml:199]: also need to add shit to el9-looksasides…

**Comment by ekohl** [comment]
> smart_proxy_pulp needs

We already have Ruby 3.0 & 3.1 testing in Jenkins, so I'd argue it's not needed.

**Comment by evgeni** [comment]
> > smart_proxy_pulp needs
> 
> We already have Ruby 3.0 & 3.1 testing in Jenkins, so I'd argue it's not needed.

If you look at the PR, it also adjusts the gemspec to allow 3.0, I can submit that as an own PR though.

File: package_manifest.yaml [hunk 2]
```diff
@@ -870,10 +877,16 @@ ruby_plugin_packages:
...
+  vars:
+    copr_projects:
+      - "{{ hostvars['plugins-copr-with-el9'] }}"
   hosts:
     rubygem-faraday_middleware: {}
+    rubygem-httparty: {}
     rubygem-infoblox: {}
     rubygem-logify: {}
+    rubygem-mqtt: {}
+    rubygem-multi_xml: {}
```
Comment (ekohl) [inline package_manifest.yaml:889]: These look correct: they're only mentioned in the plugin comps.

**Review by ekohl — APPROVED** [review]

**Comment by ekohl** [comment]
> And there is no Jenkins run in the PR?

Are you confusing it with the `smart_proxy_container_gateway` PR? I was talking about `smart_proxy_pulp`.

**Pull request merged**